src/assets/stylesheets/Sidebar.scss

```diff
@@ -80,7 +80,7 @@
 }
 
 .sidebar__bar-option-wrapper {
-  padding: 0.625rem;
+  padding: calc(0.75 * $space-0-5);
```

Comment:
cool! I did wonder if using something like `min(1px, 0.625rem)` would work but your approach will be more consistent across screens and devices